jscomp/flow.ml

```diff
@@ -0,0 +1,117 @@
+open Types
+
```

Comment:
a minor style issue, in general, open is not encouraged, especially for modules like `Types` which is an internal module